cpu/stm32_common/Makefile.include

```diff
@@ -30,11 +30,11 @@ export CFLAGS += -D$(CPU_LINE) -DCPU_LINE_$(CPU_LINE)
 export CFLAGS += -DSTM32_FLASHSIZE=$(FLASHSIZE)U
 
 info-stm32:
-	@$(COLOR_ECHO) "CPU: $(CPU_MODEL)"
-	@$(COLOR_ECHO) "\tLine: $(CPU_LINE)"
-	@$(COLOR_ECHO) "\tPin count:\t$(STM32_PINCOUNT)"
-	@$(COLOR_ECHO) "\tROM size:\t$(ROM_LEN) ($(FLASHSIZE) Bytes)"
-	@$(COLOR_ECHO) "\tRAM size:\t$(RAM_LEN)"
+	@echo -e "CPU: $(CPU_MODEL)"
```

Comment:
There is a reason why this `COLOR_ECHO` variable was introduced. AFAIK (but please research yourself as well in the git history), there are `echo` implementations that don't support the `-e` parameter, so there needs to be a replacement for that.

Reply:
Thanks for the info. I cannot find the PR, though.

I found [this post about OSX's ech](https://superuser.com/questions/33914/why-doesnt-echo-support-e-escape-when-using-the-e-argument-in-macosx). It says it does support "-e" for things like `\t` and `\n` but not for ANSI escape codes.
If this is the case, then my change is actually better: one can use `-e` even if colors are not supported.

I'd appreciate if a mac user tried it, though.

Reply:
I know for a fact that with most simple shells like dash it just prints `-e string` if you call `echo -e "string"` ;-).

The PR isn't really easy to track, but some investigation later I was able to track it down to 351dea1dbd5d5b252d2e089f303f712e865f89c3, so #620 ;-). Doesn't give much info though, why `ECHO` was introduced as a variable though :-/. But I guess the shell executor on Jenkins was exactly doing what I described above ;-).